backend/layers/persistence/persistence.py

```diff
@@ -45,27 +46,31 @@
 
 
 class DatabaseProvider(DatabaseProviderInterface):
-    def __init__(self, database_uri: str = None, schema_name: str = "persistence_schema") -> None:
+    def __init__(self, database_uri: str = None, schema_name: str = SCHEMA_NAME) -> None:
         if not database_uri:
             database_uri = CorporaDbConfig().database_uri
         self._engine = create_engine(database_uri, connect_args={"connect_timeout": 5})
         self._session_maker = sessionmaker(bind=self._engine)
+        self._schema_name = schema_name
         try:
-            self._create_schema(schema_name)
+            self._create_schema(self._schema_name)
```

Comment:
remove arg here, now that _create_schema method signature has changed

Reply:
```suggestion
            self._create_schema()
```